src/tests/ftest/control/dmg_storage_query.py

```diff
@@ -60,6 +61,7 @@ def check_dev_state(self, devs_info, state):
         if err:
             self.fail("Found device(s) in bad state: {}".format(err))
 
+    @skipForTicket("DAOS-4286")
```

Comment:
DAOS-4286 is **Resolved**.  You can't skip a test for a ticket that has been resolved.

Reply:
@Brian Thanks for the review and comment, the failures should be DAOS-5320.